src/common/Input/Input.jsx

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './Input.css'; // Using CSS Modules for scoped styles
 
-const Input = ({ labelText, placeholderText, onChange }) => {
+const Input = ({ labelText, type, value, onChange, placeholderText, required }) => {
```

Comment:
You could make use of the `{...props}` passing here similarly to how you did with the Button